Explain how to programmatically upload artifacts, close MISC-125 (#26)
gatling/frontline-cloud-doc: content/tutorials/bundle/index.md

## Upload the artifact to FrontLine

### Option 1: Manual Upload

{{< include upload_manual.md "target/artifact.jar" >}}

Upload it to FrontLine, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.

{{< img src="choose_artifact.png" alt="Choose the generated artifact" >}}

> You now have to click on the Upload button to upload it to FrontLine. After a few seconds, the upload will be complete, and our artifact will be successfully ready to use!

Reviewer: ```suggestion
You now have to click on the Upload button to upload it to FrontLine. After a few seconds, the upload will be complete, and your artifact will be ready to use!
```
